src/patients/view/AddNewPatient.tsx

```diff
@@ -0,0 +1,38 @@
+import { Button, Icon, Typography } from '@hospitalrun/components'
+import React from 'react'
+import { useHistory } from 'react-router-dom'
+
+import useTranslator from '../../shared/hooks/useTranslator'
+
+const AddNewPatient = () => {
```

Comment:
Thoughts on `NoPatientsExist` for this component name? `AddNewPatient` could potentially be confusing with `NewPatient` component.